src/Components/Directory.Build.targets

```diff
@@ -9,7 +9,7 @@
 
     <_BlazorDevServerPath>$(ArtifactsDir)bin/Microsoft.AspNetCore.Components.WebAssembly.DevServer/$(Configuration)/$(DefaultNetCoreTargetFramework)/blazor-devserver.dll</_BlazorDevServerPath>
     <RunCommand>dotnet</RunCommand>
-    <RunArguments>exec &quot;$(_BlazorDevServerPath)&quot; serve --applicationpath &quot;$(TargetPath)&quot; $(AdditionalRunArguments)</RunArguments>
+    <RunArguments>exec &quot;$(_BlazorDevServerPath)&quot; --applicationpath &quot;$(TargetPath)&quot; $(AdditionalRunArguments)</RunArguments>
```

Comment:
Is this the only place we use serve in our build/tests?  If so I can do a follow up PR getting rid of the custom serve stuff I added in helix for serve 🎈

Reply:
Yeah, outside of the build targets that ship in the targets, this is the only other place this command is used.